tools/clang/test/HLSLFileCheck/hlsl/semantics/legacy/obsolete_semantics_ps_3.hlsl

```diff
@@ -0,0 +1,8 @@
+// RUN: %dxc -E main -T ps_6_0 %s -Gec | FileCheck %s
```

Comment:
If we're keeping the Gec conversion, there should be a separate test for that which checks for the warning that the conversion has taken place  `warning: DX9-style semantic "VFACE" mapped to DX10 system semantic "SV_IsFrontFace" due to -Gec flag. This functionality is deprecated in newer language versions.`

Either way, this test should not include the `-Gec` flag since it will either prevent the warning checked here or else it will be unneeded. Additionally, when there is no error, warnings go into stderr, which means that FileCheck needs the flag `-input=stderr`. As would the hypothetical second test for the conversion warning.

Reply:
I just removed `-Gec`. Sorry for making you confused.